internal/command/android.go

```diff
@@ -41,8 +46,10 @@ func (cmd *Android) Parse(args []string) error {
 
 	flags := &androidFlags{
 		CommonFlags: commonFlags,
+		TargetArch:  &targetArchFlag{string(ArchMultiple)},
 	}
 
+	flagSet.Var(flags.TargetArch, "arch", fmt.Sprintf(`List of target architecture to build separated by comma. Supported arch: %s.`, androidArchSupported))
```

Comment:
Is it better to have one more `-arch` option? How about, for example `-os android/amd64` where `-os` consumes `OS/ARCH`?

It can also parse multiple targets, such as: `-os android/386,amd64,arm,arm64`, or just `-os android/universal`.

Reply:
That makes sense. The choice to have two options `arch` and `os` was added to follow the Go `GOARCH` and `GOOS`. We'd could change them but could introduce a potential breaking change.
Also please note that the fyne cli and the fyne-cross cli are slightly different and would great if in the future we'd align them.